pyproject.toml

```diff
@@ -46,7 +46,9 @@ dependencies = [
     "gitpython>=3.1.29",
     "pytz>=2022.6",
     "giturlparse>=0.10.0",
-    "orjson>=3.6.7,<4,!=3.9.11,!=3.9.12,!=3.9.13,!=3.9.14,!=3.10.1 ; platform_python_implementation != 'PyPy'",
+    "orjson>=3.6.7,<4,!=3.9.11,!=3.9.12,!=3.9.13,!=3.9.14,!=3.10.1 ; platform_python_implementation != 'PyPy' and sys_platform != 'emscripten'",
```

Comment:
this is here on purpose because you've branched off of it?

Reply:
I'm separating `emscripten` (WASM) orjson dependency from all other system and architectures. pyodide has only 3.10.1 available which is buggy and crashes so I do not allow it in production. it will be good enough for WASM